.travis.yml

```diff
@@ -46,7 +46,8 @@ before_script:
 script:
   # mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent test coveralls:report -e
   # Skip coveralls temporarily, fix it asap
-  - mvn clean test
+  - mvn clean test -q
+  - mvn clean test -q -Psandbox -Pspark3
```

Comment:
Why needs to add these two commands? remove the first one?

Reply:
The former is for spark24 which is the default profile, the latter is for spark31. We should make sure that test cases are passed in both versions.

Reply:
Got it.